src/PKSim.Core/Services/ModuleSnapshotSerializer.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Text;
+using OSPSuite.Core.Services;
+using PKSim.Core.Snapshots;
+
+namespace PKSim.Core.Services;
+
+public interface IModuleSnapshotSerializer
+{
+   public string Serialize(Project project);
+}
+
+public class ModuleSnapshotSerializer : IModuleSnapshotSerializer
```

Comment:
why is this called ModuleSnapshotSerializer if it serialized a project?
Should we rename the method ?

Reply:
Yeah, I'd like to leave the name of the serializer because it's really a project that is specifically created to mirror a module. But we can say `SerializeFromProject` or something. This is a special project that is built for the module.